### PR TITLE
Define colors & normalize

### LINKED ISSUE #286: Define colors

### PR DESCRIPTION
**Pull request by dejvid15**
Closes #286 

Defined colors & typography, moved all to the bottom of the stylesheet file

### CODE REVIEW

File: app/public/css/styles.css [hunk 1]
```diff
@@ -1577,3 +1509,73 @@ td {
...
   display: flex;
 }
+
+/* REDESIGN 2.0 */
+
+/* VARIABLES */
+
+:root {
+  --pink: #e72076;
+  --darkGrey: #20323f;
```
[the remark this answers is not in the document]
Comment (darjanin) [reply app/public/css/styles.css]: Use only dashes to split long names.

File: app/public/css/styles.css [hunk 2]
```diff
@@ -1577,3 +1509,73 @@ td {
...
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: JTMarnie, Arial, Helvetica, sans-serif;
+  height: 100%;
+  color: var(--darkGrey);
+}
+
+input:focus,
```
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/public/css/styles.css]: Removed

File: app/public/css/styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/public/css/styles.css]: Will do

File: app/public/css/styles.css [hunk 3]
```diff
@@ -1577,3 +1509,73 @@ td {
...
+
+/* REDESIGN 2.0 */
+
+/* VARIABLES */
+
+:root {
+  --pink: #e72076;
+  --darkGrey: #20323f;
+  --grey: #606a71;
+  --lightGrey: #e6e6e6;
```
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/public/css/styles.css]: Ok

File: app/public/css/styles.css [hunk 4]
```diff
@@ -1577,3 +1509,73 @@ td {
...
+  --green: #78d088;
+  --blue: #78c9dd;
+  --red: #cf4040;
+  --orange: #f4b253;
+}
```
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/public/css/styles.css]: Will do

File: app/public/css/styles.css [hunk 5]
```diff
@@ -1577,3 +1509,73 @@ td {
...
+  height: 100%;
+  font-size: 14px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: JTMarnie, Arial, Helvetica, sans-serif;
+  height: 100%;
+  color: var(--darkGrey);
```
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/public/css/styles.css]: Will refactor

**Comment by dejvid15** [comment]
@darjanin Rebased buttons and added `light & dark` variations for `brand` colors & used in buttons. Also implemented the rest of your review comments.

**Pull request merged**